.github/workflows/go.yml

```diff
@@ -288,12 +288,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [1.17, 1.18]
+        go: [1.19, '1.20']
         include:
```

Comment:
These include bits are not really necessary, we can just add static check as another matrix element as we want to check all the combinations (2 versions of go each with the same static check). So there is no need to explicitly list the combinations.

Reply:
(or read staticcheck version from env as kou suggested and just have the go array.

Reply:
updated!